sql/1.145/maintainer_gh.pl

```diff
@@ -14,10 +14,10 @@
         my %updated_maintainer;
         my $duckco_user = $d->rs('User')->find({username => $maintainer});
 
-        if ($duckco_user) {
+        if ($duckco_user || (!$d->is_live)) {
```

Comment:
Outside of the `while` loop, i.e. line 11:
```Perl
my $is_live = $d->is_live;
```
Then use the variable.

Won't this set the `duckco` maintainer to a `github` maintainer if the user doesn't exist?